Makefile

```diff
@@ -6,7 +6,7 @@ lint:
 
 ## Go test
 test: lint
-	CGO_ENABLED=0 go test -timeout 1m -v -covermode=atomic ./...
+	CGO_ENABLED=0 go test -tags testtools -timeout 2m -v -covermode=atomic ./...
```

Comment:
I guess we have long tests? Can you list them and drop a KTLO Jira for them so we don't lose track. Having long tests means people won't run them.

Reply:
mhmm, i think i might not need that anymore. I was seeing something weird and added the 2m. I can no run it without the 2m. Ill revert that.